client/signup/steps/design-type/index.jsx

```diff
@@ -11,27 +13,39 @@ import SignupActions from 'lib/signup/actions';
 import analytics from 'lib/analytics';
 import Card from 'components/card';
 
-export default React.createClass( {
-	displayName: 'DesignType',
+import BlogImage from '../design-type-with-store/blog-image';
+import PageImage from '../design-type-with-store/page-image';
+import GridImage from '../design-type-with-store/grid-image';
+
+class DesignTypeStep extends Component {
+	static propTypes = {
+		translate: PropTypes.func
+	};
+
+	static defaultProps = {
+		translate: identity
+	};
 
 	getChoices() {
+		const { translate } = this.props;
+
 		return [
-			{ type: 'blog', label: this.translate( 'A list of my latest posts' ), image: <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 310 230"><rect x="15" y="15" fill="#E8F0F5" width="280" height="70"/><rect x="15" y="98" fill="#C3EF96" width="194" height="85"/><rect x="15" y="195" fill="#C3EF96" width="194" height="35"/></svg> },
-			{ type: 'page', label: this.translate( 'A welcome page for my site' ), image: <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 310 230"><rect fill="#E8F0F5" width="310" height="110"/><rect x="114" y="205" fill="#E8F0F5" width="82" height="25"/><rect x="15" y="205" fill="#E8F0F5" width="82" height="25"/><rect x="213" y="205" fill="#E8F0F5" width="82" height="25"/><rect x="15" y="36" fill="#D2DEE6" width="153" height="13"/><rect x="15" y="59" fill="#D2DEE6" width="113" height="13"/><rect x="15" y="82" fill="#C3EF96" width="30" height="13"/><rect x="15" y="125" fill="#C3EF96" width="280" height="65"/></svg> },
-			{ type: 'grid', label: this.translate( 'A grid of my latest posts' ), image: <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 310 230"><rect x="15" y="15" fill="#E8F0F5" width="280" height="40"/><rect x="114" y="70" fill="#C3EF96" width="82" height="65"/><rect x="15" y="70" fill="#C3EF96" width="82" height="65"/><rect x="213" y="70" fill="#C3EF96" width="82" height="65"/><rect x="114" y="150" fill="#C3EF96" width="82" height="65"/><rect x="15" y="150" fill="#C3EF96" width="82" height="65"/><rect x="213" y="150" fill="#C3EF96" width="82" height="65"/></svg> },
+			{ type: 'blog', label: translate( 'A list of my latest posts' ), image: <BlogImage /> },
+			{ type: 'page', label: translate( 'A welcome page for my site' ), image: <PageImage /> },
+			{ type: 'grid', label: translate( 'A grid of my latest posts' ), image: <GridImage /> },
 		];
-	},
+	}
 
-	renderChoice( choice ) {
+	renderChoice = ( choice ) => {
```

Comment:
Instead of writing

```js
renderChoice = ( choice ) => {
    return (
        // ...
    );
}
```

you could simplify it to

```js
renderChoice = ( choice ) => (
    // ...
);
```